tests/networking/parameters/parameters.go

```diff
@@ -33,11 +33,8 @@ var (
 	CertsuiteUndeclaredContainerPortsUsageTcName = "networking-undeclared-container-ports-usage"
 	CertsuiteReservedPartnerPortsTcName          = "networking-reserved-partner-ports"
 	CertsuiteDualStackServiceTcName              = "networking-dual-stack-service"
-	CertsuiteDpdkCPUPinningExecProbe             = "networking-dpdk-cpu-pinning-exec-probe"
-
-	DpdkPodName                   = "dpdk-pod"
-	NetworkingTestSkipLabel       = map[string]string{"redhat-best-practices-for-k8s.com/skip_connectivity_tests": ""}
-	NetworkingTestMultusSkipLabel = map[string]string{"redhat-best-practices-for-k8s.com/skip_multus_connectivity_tests": ""}
+	NetworkingTestSkipLabel                      = map[string]string{"redhat-best-practices-for-k8s.com/skip_connectivity_tests": ""}
+	NetworkingTestMultusSkipLabel                = map[string]string{"redhat-best-practices-for-k8s.com/skip_multus_connectivity_tests": ""}
```

Comment:
@sebrandon1 How did this pass linting before? It looks like someone manual adjusted the formatting, which isn't great, can we just update the linter? Otherwise every PR in this file has to be manually edited.